technology/coding_style.md

```diff
@@ -2,19 +2,23 @@
 
 ## Standard JS
 
-Coding style tends to be hard to agree on and leads to a lot of bikeshedding. We decided to outsource our decisions to the [standard](https://standardjs.com/) JS coding style. It has nothing standard but its name, but it aligns everyone on the same coding style.
+Coding style tends to be hard to agree on and leads to a lot of bikeshedding. Thus we decided to embrace the [JavaScript Standard Style](https://standardjs.com/) style guide. It has nothing standard but its name, but it aligns everyone on the same coding style.
```

Comment:
Important - 'Standard' is the name of the package, but it is (ironically) NOT the golden standard of JS coding style.

I suggest rephrasing accordingly.